internal/appsec/internal/intake/api/api.go

```diff
@@ -0,0 +1,361 @@
+package api
```

Comment:
I think it's safe to now move everything from within `internal/appsec/internal/*` one level up to `internal/appsec/`, don't you think? It might be a bit overly ambitious and strenuous if we have to start using the concept of privacy within an already private module.

Reply:
I like making internal things internal to avoid leaking internal things out of the package being defined, no matter where they are. So we indeed possibly end up with nested internal packages.

Reply:
I think it makes sense if you want that API to not be used by other contributors to this repository, except of course the ones working on `internal/appsec/`. Fine to resolve by me.